.pre-commit-config.yaml

```diff
@@ -4,12 +4,12 @@ repos:
       hooks:
           - id: black
             language_version: python3.9
-    - repo: https://github.com/asottile/add-trailing-comma
```

Comment:
Is this functionality part of isort or something?

Reply:
No, but I was trying to be more consistent with bedrock which doesn't have it. If it is preferred I can keep it in there and consider adding it to bedrock.

Reply:
I think it used to be in bedrock. Maybe black does it now? I'm not sure.